issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerModel.kt

```diff
@@ -8,7 +8,9 @@
 
 package com.adyen.checkout.issuerlist
 
+import com.adyen.checkout.components.ui.adapter.SimpleTextListItem
+
 data class IssuerModel(
     val id: String,
     val name: String
-)
+) : SimpleTextListItem(name)
```

Comment:
This shouldn't be the case either. For issuers we shouldn't use `SimpleTextListAdapter`